src/platforms/generic_unix/mapped_file.c

```diff
@@ -34,12 +34,14 @@ MappedFile *mapped_file_open_beam(const char *file_name)
 {
     MappedFile *mf = malloc(sizeof(MappedFile));
     if (IS_NULL_PTR(mf)) {
+        fprintf(stderr, "Unable to allocate MappedFile struct\n");
```

Comment:
Feel free to veto.  I found it useful when misconfigured tests failed.

Reply:
I agree, looks good.